suitcase.py

```diff
@@ -39,34 +40,30 @@ def export(headers, filename):
                 desc_group.create_dataset('event_times', event_times)
                 data_group = desc_group.create_group('data')
                 ts_group = desc_group.create_group('timestamps')
+                [fill_event(e) for e in events]
                 for key, value in data_keys.items():
+                    print('data key = %s' % key)
                     timestamps = [e['timestamps'][key] for e in events]
                     ts_group.create_dataset(key, data=timestamps)
-                    for event in events:
-                        fill_event(event)  # fill any external data, in place
                     data = [e['data'][key] for e in events]
                     dataset = data_group.create_dataset(key, data=data)
                     # Put contents of this data key (source, etc.)
                     # into an attribute on the associated data set.
-                    value['data_broker_shape'] = value.pop('shape')
-                    value['data_broker_dtype'] = value.pop('dtype')
-                    print(value)
-                    _safe_attrs_assignment(dataset, value)
+                    # value['data_broker_shape'] = value.pop('shape')
```

Comment:
Commenting these lines out was an "emergency" measure to get things running. (I doubt the visiting users will say, "Hey, where are they data keys?!") But we should fix it this right way.